faucet/valve.py

```diff
@@ -899,11 +901,21 @@ def _lacp_actions(self, lacp_pkt, port, now):
                     down_state = lacp_peer
                 elif lacp_peer.dyn_lldp_beacon_recv_state == STACK_STATE_FAILOVER:
                     failover_state = lacp_peer
+                lacp_peer_dp = None
+                if lacp_peer.lldp_peer_mac in self.dp.dps_by_mac:
+                    lacp_peer_dp = self.dp.dps_by_mac[lacp_peer.lldp_peer_mac]
+                if lacp_peer_dp:
```

Comment:
why is this if statement necessary?  Seems like you can just keep this behing the first if, and remove the lacp_peer_dp = None assignment